pkgs/tools/misc/grub/2.0x.nix

```diff
@@ -53,6 +55,13 @@ stdenv.mkDerivation rec {
     ./fix-bash-completion.patch
   ];
 
+  postPatch = if kbdcompSupport then ''
+    sed -i util/grub-kbdcomp.in -e 's@\bckbcomp\b@${ckbcomp}/bin/ckbcomp@'
+  '' else ''
+    echo '#! ${stdenv.shell}' > util/grub-kbdcomp.in
```

Comment:
Also you might need `runtimeShell` or something, to work properly with cross-compilation. cc @lopsided98

Reply:
Yes, `runtimeShell` is needed here because this script will need to run on the host platform.

Reply:
Changed, I don't know how to test it with cross-compilation though.

Reply:
Fyi @mdevlamynck it's as easy as e.g. `pkgsCross.aarch64-multiplatform.grub2`. And if you don't have a non-x86 machine for testing you can "cross-compile" to 32-bit and test locally with `pkgsCross.gnu32` :)